app.rb

```diff
@@ -0,0 +1,138 @@
+require './person'
+require './capitalize_decorator'
+require './trimmer_decorator'
+require './student'
+require './classroom'
+require './teacher'
+require './rental'
+require './book'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'There are no books to show! Please add a book.' if @books.empty?
+
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+    puts
+    puts
+  end
+
+  def list_all_people
+    puts 'There are no people to show! Please add a student or a teacher.' if @people.empty?
+
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, Id: #{person.id}, Age: #{person.age}" }
+    puts
+    puts
+  end
+
+  def create_a_person
+    print 'Do you want to create a student (1) or teacher (2)? [Input the number]: '
+    choice = gets.chomp
+
+    case choice
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input! Please enter 1 or 2'
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp.downcase
+
+    parent_permission = permission == 'y'
+
+    student = Student.new('Grade 12', age, name, parent_permission)
+    @people << student
+
+    puts 'Person/Student created successfully'
+    puts
+    puts
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Specialization: '
+    specialization = gets.chomp
+
+    teacher = Teacher.new(specialization, age, name)
+    @people << teacher
+
+    puts 'Person/Teacher created successfully'
```

Comment:
- [ ] Also, trying to create a teacher data wasn't successful as well as seen in the screenshot below
![Screenshot from 2023-03-10 12-51-08](https://user-images.githubusercontent.com/46388113/224319644-23fe2fa2-a61a-40d7-b90b-3f0f147733e1.png)